src/analyses/goto_check.cpp

```diff
@@ -2453,3 +2492,48 @@ void goto_check(
   const namespacet ns(goto_model.symbol_table);
   goto_check(ns, options, goto_model.goto_functions, message_handler);
 }
+
+void goto_checkt::add_active_named_check_pragmas(
+  source_locationt &source_location)
+{
+  for(auto it : name_to_flag)
```

Comment:
I'd avoid "it" as a name for this isn't an iterator. Also, I'm never quite sure what the exact type here is, so I tend to use `const auto &entry : name_to_flag` to make sure it's a const reference and not a copy.